infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutKmPostController.kt

```diff
@@ -117,9 +117,9 @@ class LayoutKmPostController(
     fun validateKmPost(
         @PathVariable("publishType") publishType: PublishType,
         @PathVariable("id") id: IntId<TrackLayoutKmPost>,
-    ): ValidatedAsset<TrackLayoutKmPost> {
+    ): ResponseEntity<ValidatedAsset<TrackLayoutKmPost>> {
```

Comment:
Muutin kaikki publicationservicen funkkarit toimimaan listalla id:tä, ihan yhtenäisyyden vuoksi ja koska se on nykyisellä kakutusrakenteella ihan nopsaa. En kuitenkaan muuttanut näitä controller-metodeita etukenossa koska käyttöpaikkaa ei vielä ole.